Tests/AK/TestTime.cpp

```diff
@@ -274,3 +274,167 @@ TEST_CASE(is_negative)
     EXPECT_EQ(result.to_nanoseconds(), 5);
     EXPECT(!result.is_negative());
 }
+
+struct YearAndDays {
+    int year;
+    int days;
+};
+
+TEST_CASE(years_to_days_since_epoch_points)
+{
+    auto test_data = Array {
```

Comment:
nit: I think if you type out the, uh, type, here on the lhs you don't need to repeat it on every line on the rhs I think.

Reply:
I didn't do that while writing the code because then I'll have to hard-code the number "15". But you're right, the committed version should be written this way.

Reply:
This is going to be disgusting, because the compiler wants an initializer_list of exactly one item, namely the array. The way to write that is:
```
    Array<YearAndDays, 15> test_data = { { // Oof
        { 1969, -365 },
        { 1970, 0 },
        { 1971, 365 },
        { 1900, -25567 },
        { 2023, 19358 },
        { 1800, -62091 },
        { 2100, 47482 },
        { 0, -719528 },
        { 4000, 741442 },
        { -10000, -4371953 },
        { 10000, 2932897 },
        { -1000000, -365962028 },
        { 1000000, 364522972 },
        { -5877640, -2147483456 },
        { 5881474, 2147444740 },
    } }; // Oof
```
However, that sounds like it is an issue for another time.